### PR TITLE
Use Parliamentary Members API to get MP details

### PR DESCRIPTION
**Pull request by lozette**
## Changes

We want to be able to show the name and contact details of a local authority's MP in the application, so that users can send a letter (or other correspondence) to the MP and notify them of a school's academisation.

The Parliamentary Members API is available to find details about members of parliament. 

Build a new MembersApi Client. Use it to find a constituency via a free text search (we will use the name of a school district as the search term). Then use that constituency data to find the member of parliament's name and contact details.

Render the MP's name and contact details as custom models we can later use in the UI.

## Checklist

- [x] Attach this pull request to the appropriate card in Trello.
- [x] Update the `CHANGELOG.md` if needed.

### CODE REVIEW

**Comment by lozette** [comment]
[Fetch MP details from the members API](https://trello.com/c/sCIdfDJm/1427-fetch-mp-details-from-the-members-api)

**Review by mec — CHANGES_REQUESTED** [review]
Couple of comments in there, looks like a solid start though! 👍

File: app/models/members_api/client.rb [hunk 1]
```diff
@@ -1,10 +1,42 @@
...
+      Result.new(body, nil)
+    else
+      Result.new(nil, Error.new(I18n.t("members_api.errors.other")))
+    end
+  end
+
+  def member_id(search_term)
+    constituency_data = constituency(search_term)
+    if constituency_data.object["items"].count > 1
+      Result.new(nil, Error.new(I18n.t("members_api.errors.multiple", search_term: search_term)))
```
[the remark this answers is not in the document]
Comment (lozette) [reply app/models/members_api/client.rb:29]: Yes, oops!

File: app/models/base_api_model.rb [hunk 2]
```diff
@@ -1,4 +1,4 @@
-class AcademiesApi::BaseApiModel
+class BaseApiModel
```
[the remark this answers is not in the document]
Comment (lozette) [reply app/models/base_api_model.rb]: It's made this commit hyooooge but it's done

**Review by mec — APPROVED** [review]

**Pull request merged**